errorHandler.js

```diff
@@ -0,0 +1,53 @@
+const { HttpCodes } = require("./messages/httpCodes");
+const { customErrors } = require("./messages/errors");
+const { messages } = require("./messages/messages");
+
+class MessageToUser {
+    constructor(name) {
+        this.statusCode = messages[name].code;
+        this.name = name;
+        this.message = messages[name].message;
+    }
+}
+
+class CustomError {
+    constructor(name) {
+        this.statusCode = customErrors[name].code;
+        this.name = name;
+        this.message = customErrors[name].message;
+    }
+}
+
+function errorHandler(err) {
+    let handledError = {
+        statusCode: null,
+        message: null,
+        type: null,
+    };
+
+    if (err instanceof MessageToUser) {
```

Comment:
I am not sure that this scenario is possible. We have any scenario where we put Successful message inside of `errorHandler` function?

Reply:
> We have any scenario where we put Successful message inside of errorHandler function?

Yes, we have. The logic of successful messages will be transferred to another file.

Reply:
> Yes, we have. The logic of successful messages will be transferred to another file.
Sorry, we don't. This scenario will be removed.